src/components/DataControl/index.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import Sliders from '../Sliders';
+
+class DataControl extends Component {
+  static propTypes = {
+    timeRange: PropTypes.number.isRequired,
+  };
+
+  handleSlidersChange = (event, value) => {
+    // eslint-disable-next-line react/prop-types
+    const { history } = this.props;
+    history.push(`?numDays=${value}`);
+  };
+
+  render() {
+    const { timeRange } = this.props;
+    return (
+      <div>
+        <Sliders
```

Comment:
We don't know if we're going to have more than 1 slider in the future. We might have checkboxes.
No need to put the `Slider` inside of `Sliders`.
Put `Slider` directly in here.